modules/lm-coursier/src/main/scala/lmcoursier/definitions/Authentication.scala

```diff
@@ -1,12 +1,14 @@
 package lmcoursier.definitions
 
-import dataclass.data
+import dataclass._
 
 @data class Authentication(
   user: String,
   password: String,
   optional: Boolean = false,
-  realmOpt: Option[String] = None
+  realmOpt: Option[String] = None,
+  @since
+  headersOpt: Option[Seq[(String,String)]] = None
```

Comment:
Let's make this one a simple `Seq`, like `headers: Seq[(String, String)] =  Nil`. This class is supposed to be a simple clone of [`coursier.core.Authentication`](https://github.com/coursier/coursier/blob/796767f765f5ef7211252370e76fcfb1882734d7/modules/util/shared/src/main/scala/coursier/core/Authentication.scala).